src/applications/vaos/utils/address.js

```diff
@@ -18,6 +22,25 @@ export function distanceBetween(lat1, lng1, lat2, lng2) {
   return parseFloat((R * c).toFixed(1));
 }
 
+export function calculateBoundingBox(lat, long, radius) {
```

Comment:
Is this borrowed from Facility Locator?  Might be worth adding some comments as to how it works or a link to the resource it's from.

Reply:
Added. It's adapted from a library a found in npm. I chose not to use the bounding box calc from the facility locator, since this is slightly more accurate and not much more complicated (just some math for dealing with longitude).